tests/tasks/cubejs/test_cubejs_tasks.py

```diff
@@ -129,7 +135,11 @@ def test_run_with_max_wait_time_raises(self):
                 api_secret="foo",
                 query="query",
                 security_context={"foo": "bar"},
-                max_wait_time=15,
+                wait_time_between_api_calls=1,
+                max_wait_time=3,
             )
 
-        assert "Cube.js load API took too long to provide a response." in str(exc)
+        assert (
+            "Cube.js load API took longer than 3 seconds to provide a response."
+            in str(exc)
+        )
```

Comment:
You can use the `match` parameter of `pytest.rasises` for to eliminate these lines:
```
with pytest.raises(FAIL, match="Cube.js load API took longer than 3 seconds to provide a response.") as exc:
```